attributes/default.rb

```diff
@@ -15,10 +15,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 #
-
+srv_provider = ::Chef::Platform::ServiceHelpers.service_resource_providers
```

Comment:
We have an ohai attribute for the init package that's probably easier to use here